### ISSUE TITLE
`variable_active_storage_on_off` loops over wrong ref

### ISSUE THREAD

**Issue by pseudocubic** [lanl-ansi/PowerModelsMLD.jl#15]
There is a bug in `variable_active_storage_on_off`, the symbol `:gen` is used instead of `:storage`, and storage also has no `pmax`, so the upper and lower bounds will have to be determined some other way, like with this expression found in `variable_active_storage`

```julia
inj_lb, inj_ub = ref_calc_storage_injection_bounds(ref(pm, nw, :storage), ref(pm, nw, :bus), cnd)
```

This only works currently because there are also active generators in the test case. If for example generator 1 is deactivated, then the model creation will fail because key 1 won't exist.

CC @noahrhodes

**Comment by ccoffrin** [comment]
So right, my bad for missing that!

**Comment by noahrhodes** [comment]
@pseudocubic I believe this was fixed

**Comment by pseudocubic** [comment]
@noahrhodes you're right

Closed by #16
